### ISSUE TITLE
[Bugs] Another list with stuff

### ISSUE THREAD

**Issue by Kimod** [Drutol/MALClient#53]
I know you are working hard on something complicated, so I prepared a little list for you so you can take a little break :p

* Calendar: Reload doesn't respect "Start on Summary" when you reload - http://puu.sh/qPzjo.gif
 * :heavy_check_mark: Fixed + fixed building calendar bug.
* Scrollbar in Calendar isn't at the edge (this has nothing to do with that previous scrollbar bug)
 * :heavy_check_mark: Never ending story ね？
* Top manga always shows amount of volumes even if you have chapters selected - http://puu.sh/qJb0e.png
 * :heavy_check_mark: As it turns out I wast too quick with copy pasting and forgot that I don't really have volume data from top anime so they are always going to display this value. + details will now switch volumes/chapters purely for consistency as these values are already visible ^^
* In manga details, "7" - http://puu.sh/qPyKo.png is the amount of Volumes, not Chapters
 * :heavy_check_mark:  Oh, and I thought I spotted someting...
* No stock photo for Recoms in Manga details - http://puu.sh/qJbyR.png
 * :heavy_check_mark:  I was wondering how long will this one take you to notice ^^. They had malformed html there but things seem to be okay now.
* If you click on Seasonal Anime, then on Recommendations, then on Settings & More (or on anime in Recomms), and close it with that swipe, "Recommendations" changes to "Summer 2016 - All" - http://puu.sh/qJgfW.gif
 * :heavy_check_mark: This was interesting one :)
* Ok, since you figured out that one above this one, I will give you similar - After a fresh start, if you click in profile or history on anime and then in "Related" on manga, it will change Profile - name/History - name to name - Watching - http://puu.sh/qTHUJ.gif
 * :heavy_check_mark: Fixed
* When you first click on "Seasonal Anime", it shows "Summer 2016 - All", but if you click on it again later, it will show "Airing - All"
 * :heavy_check_mark: Fixed
* You shouldn't show "Index" in Seasonal Anime / Top anime, because you can't sort that list, so it is pointless
 * :heavy_check_mark: Fixed
* Typo - "Decsending" should be "Descending" - http://puu.sh/qJia3.png
 * :heavy_check_mark: Fixed
* When I enter number and click on Enter (on keyboard), it says "Wrong Input!", could you change this so it will click on that checkmark or is this some kind of element limitation? - http://puu.sh/qJiza.gif
 * :heavy_check_mark: Fixed
* "Airing" in that menu should be 'All' - http://puu.sh/qJjaf.png
 * :heavy_check_mark: Fixed
* Select season doesn't work? - http://puu.sh/qPz0V.gif
 * :white_flower: Guess what! Microsoft has bugged combobox in anniversary update, you can use scroll wheel to select item but not mouse. ¯\_(ツ)_/¯ I'd have to make anniversary update minimum targeted version of sdk in order to fix this and I don't really want to this yet. I don't know what to do with this.https://social.msdn.microsoft.com/Forums/en-US/ae9d0b92-5c97-40cf-9c39-fa940121b2b7/uwp14393combobox-within-flyout-loss-mouse-touch-interaction-after-target-the-uwp-project-to?forum=wpdevelop
 * :heavy_check_mark: After some hacking I got this working but I;m still annoyed by this behavior.
* Episodes: "0" should be '?' in recoms - http://puu.sh/qJjst.jpg
 * :heavy_check_mark: Fixed
* Should I report all Synopsis things or just don't bother with them? - http://puu.sh/qJkCJ.png
 * :white_flower: Yeah report them and eventually we will get rid of most of these ^^. I did some fixes here already.
* When you short shows by Score, could you then sort them by Title? ^^ So your 10/10 should go abc>, then your 9/10 should go abc> http://puu.sh/qPr3F.png
 * :heavy_check_mark: Done
* One more Recommendations scaling bug - http://puu.sh/qPq97.gif
 * :heavy_check_mark: Fixed
* When you click on friends profile, then on your friend's friend profile, then click on history (or anything else in hamburger menu that doesn't delete back button) and then press that back button, it goes to your friends profile, so it skips your friend's friend - http://puu.sh/qPqja.gif 
 * :heavy_check_mark: Fixed
* Search magnifier in Anime list disappears, if you click on "Account" in settings - http://puu.sh/qPqNs.gif
 * :heavy_check_mark: Fixed
* I swear I only pinned it once… :D (pinned posts multiply in forums) - http://puu.sh/qPryG.gif      
 * :heavy_check_mark: It was very cleverly doubling xd
* If a person doesn't have Voiceacting/Production Roles, scrollbar is awkwardly small, it should be stretched and while we are at it, shouldn't there also be "No Voice acting roles found"? Or something like that? - http://puu.sh/qPs1X.gif
 * :heavy_check_mark: Fixed
* Words "Voiceacting" and "Voiceactors" doesn't exist :D - http://puu.sh/qPscY.jpg
 * :heavy_check_mark: Nah they do... really believe me...
* No scrollbar in character details - http://puu.sh/qPsAa.jpg
 * :heavy_check_mark: Fixed
* If you try to delete entry you don't have in list, it crashes the app - http://puu.sh/qYBE5.gif
 * :heavy_check_mark: Fixed
* Some shows have their airing day still after they are done airing - http://puu.sh/quioF.png , you could probably somehow check for "Status" and if it's Airing > show airing day and if it's Finished Airing > don't show it
* Arrow to top disappears after you click on a show - http://puu.sh/qz2Li.gif   
 * :heavy_check_mark: Fixed
* If you search for something in anime list, then if you click on "Anime search" it searches for it, but in "Character search" it doesn't search for it - http://puu.sh/qXkxN.gif
* Couldn't you chop the empty lines in character details? - http://puu.sh/qPx7M.gif
 * :heavy_check_mark: Fixed
* There aren't any spaces between spoilers - http://puu.sh/qPxt3.png maybe have it start at a new line or just insert space?
 * :heavy_check_mark: Fixed
* When you extended one review and then clicked on different show and went to reviews, it has one review extended - http://puu.sh/qAiJG.gif
* So in a lot of places if your internet doesn't work, it will crash the app, which is always a bad thing… Or you know, when MAL went https, it was the same thing, so instead of crashing, just create a dialog, that "Something went wrong™"?
 * :heavy_check_mark: Fixed™
* If you click on a show in recommendations multiple times, it will create back arrow for each press - http://puu.sh/qSURS.gif
 * :heavy_check_mark: Fixed
* You can't select/write text in "List source", it's unusable now :( - http://puu.sh/qSBRW.png
 * :white_flower: Same thing as with season selection, bugged anniversary update... I'll consider switching target to previous W10 version.
 * :heavy_check_mark: After some hacking I got this working but I;m still annoyed by this behavior.
* If someone has comments disabled, it isn't apparent - http://puu.sh/qSUp4.png
 * :heavy_check_mark: Fixed
* In profile, if you switch Focus chapters/volumes, then in "Recent manga" grid items will change the total amount of chapters/volumes only after app restart, whereas in manga list it changes immediately - http://puu.sh/qT7vA.gif
 * :heavy_check_mark: Fixed

And since I didn't mention it anywhere, my main device is android, so when you will make something for it, I would like to be beta tester too ^^ You know… I like to find stuff :3

**Comment by Drutol** [comment]
Ufufu, after I finish moving stuff around, **everything** will require testing because 80% of code has been moved to different place/changed. Major version will jump to 5 because of it ^^

Android is developer's hell in comparison to Microsoft's way of doing things. If people knew what's behind the scenes WP would jump to at least 50% market share granted xd

**Comment by Drutol** [comment]
Btw, what are you using for mal on android? Atarashi or something, how does it compare?

**Comment by Kimod** [comment]
I use Pocket MAL, because i really don't like the layout of Atarashii... I know, it is more feature completed than Pocket MAL, but I will rather use something that is easy to use. When I use mobile to do something with MAL, I don't have to have every feature, I want it to be easy to use and look apealing ^^
I think that feature wise your app as it is now can compete with Atarashii, well maybe Atarashii has still a little more features, but I consider Atarashii really clunky, I don't know, I don't like to use it, I think it is extremly user unfriendly.

How was Hyouka? Can you agree with me that Chitanda is trash tear waifu?

**Comment by Drutol** [comment]
Hyouka itself was quite good, good enough to pick up light novel, but one-episode fillers were quite irritating.

As for Chitanda I wouldn't consider her waifu at all to be honest it's not what this show focuses on anyway.

**Comment by Kimod** [comment]
**5.0.0.0 / New**
* Seasonal anime always crashes
* When I open calendar, it will open on first day, even if I have "start on summary" selected
* When I go back on pinned post, it crashes the app - http://puu.sh/r23o5.gif
* In character details, Yato has space after his name http://puu.sh/r23FA.jpg
	
**Not Fixed**
* Why :'( - http://puu.sh/r21br.png
* Top manga always shows amount of volumes even if you have chapters selected - http://puu.sh/qJb0e.png / http://puu.sh/r223P.png
 * :heavy_check_mark: As it turns out I wast too quick with copy pasting and forgot that I don't really have volume data from top anime so they are always going to display this value. + details will now switch volumes/chapters purely for consistency as these values are already visible ^^
 * :negative_squared_cross_mark: same as before :(
*  In manga details, "7" - http://puu.sh/qPyKo.png is the amount of Volumes, not Chapters
 * :heavy_check_mark:  Oh, and I thought I spotted someting…
 * :negative_squared_cross_mark: same as before :(
* After a fresh start, if you click in profile or history on anime and then in "Related" on manga, it will change Profile - name/History - name to name - Watching - http://puu.sh/qTHUJ.gif
 * :negative_squared_cross_mark: same as before, I guess this witchery is beyond your control :/
* When you click on friends profile, then on your friend's friend profile, then click on history **(or anything else in hamburger menu that doesn't delete back button)** and then press that back button, it goes to your friends profile, so it skips your friend's friend - http://puu.sh/qPqja.gif 
 * :heavy_check_mark: Fixed
 * :negative_squared_cross_mark: Still present if you click on "Anime Search", "Recommendations", "Calendar", "Manga Search", "Articles", "News" or "Messaging"
* If someone has comments **disabled**, it isn't apparent - http://puu.sh/qSUp4.png
 * :heavy_check_mark: Fixed
 * Just so we are clear, did you read my report correctly? ^^ (I didn't mean if the person doesn't have any comments)

I can't test any bugs fixed in "Seasonal Anime", because I can't open it, so don't consider them fixed just yet! ^^ And I didn't tested mobile just yet, will probably get to it later or tomorrow...

**Comment by Drutol** [comment]
Do you have anniversary update installed?

**Comment by Kimod** [comment]
Yup, 14393.105

**Comment by Drutol** [comment]
Odd stuff is happening... seasonal is working fine, no crash after navigating back... I'll send you special slow version if increased debug capabilities ;d

**Comment by Drutol** [comment]
https://1drv.ms/u/s!Am1xQK-HVf_IkqR7WQfys2gbDouhwQ

Crash this much as you want, I should get all the logs.

**Comment by Kimod** [comment]
error 0x80073CF3, google didn't help... :/

**Comment by Drutol** [comment]
Did you uninstall previous version?

**Comment by Kimod** [comment]
Well, I tried to install it, failed, uninstalled previous version,, then I installed some stuff from "Dependencies" folder and then I was able to install it, but it crashes on startup. Is uninstall enough? Should I look for cashed data and delete that? Because I wasn't able to find any, but if I install any version, I'm always logged in

**Comment by Drutol** [comment]
It should work out of the box, I'll investigate tommorow.

**Comment by Drutol** [comment]
Ehh I have too much work to do... I'll be back tomorrow hopefully.

**Comment by Kimod** [comment]
I don't know what happened, but 5.0.2.0 no longer crashes on startup, so I'm sorry if I made you look into something that wasn't your fault ^^. Seasonal anime and going back on pinned forum post no longer crashes.

**Mobile Only**
* Show title/Character name isn't displayed - http://puu.sh/r4j59.jpg
* If I switch "Default view modes for status:" "Watching/Reading" to something different and go back to "Anime list" it crashes
* Clicking back in "Account" closes app
* Sideway scrolling in Anime details - http://puu.sh/r4jlr.jpg

**Other**
* When I open calendar, it will open on first day, even if I have "start on summary" selected
* Every person has two spaces between Birthday month and day - http://puu.sh/r4jNb.jpg
* In synopsis "naïve". "ï" is a problem - http://puu.sh/r4jW6.jpg (in app "naïve" is written "naiuml;ve")

I swear this is the last comment in this thread bringing new bugs ^^

**Comment by Drutol** [comment]
Hey, it's not as bad as I imagined!

**Comment by Kimod** [comment]
As I promised, that was last comment with bugs, creating new issue ^^
